src/main/resources/META-INF/jumpToLine.themeMetadata.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I just added this so that maybe the IDE won't complain that it doesn't know the named colors. [Here is the docs on this feature](https://jetbrains.org/intellij/sdk/docs/reference_guide/ui_themes/themes_metadata.html). Do you think it's worth having?